### PR TITLE
Fork browser prototype

### PR DESCRIPTION
**Pull request by OlegDokuka**

### CODE REVIEW

**Comment by OlegDokuka** [comment]
@kbahr, we need a release of `proteus-js` and `rsocket-rpc-js` for this changes

File: build.gradle [hunk 1]
```diff
@@ -1,34 +1,94 @@
...
+
+  group 'io.rsocket.springone.demo'
+  version '1.0.0'
+
+  sourceCompatibility = 1.8
+
+
+  ext['protobuf.version'] = '3.6.1'
+  ext['rsocket.version'] = '0.11.9'
+  ext['rsocket-rpc.version'] = '0.2.4'
```
Comment (kbahr) [inline build.gradle]: This should be 0.2.10 now

File: build.gradle [hunk 2]
```diff
@@ -1,34 +1,94 @@
...
+  ext['proteus.version'] = '1.5.0-RC'
```
Comment (kbahr) [inline build.gradle]: 1.5.2 now

File: browser-ranking/package.json [hunk 3]
```diff
@@ -12,13 +12,15 @@
...
     "lodash": "^4.17.11",
+    "micromodal": "^0.3.2",
     "node-libs-browser": "^2.1.0",
     "node-sass": "^4.9.0",
     "path": "^0.12.7",
     "proteus-js-client": "^0.1.4",
-    "proteus-js-core": "^0.0.9",
-    "proteus-js-frames": "^0.0.4",
     "rsocket-flowable": "^0.0.9",
+    "rsocket-rpc-core": "^0.0.4-BUILD-SNAPSHOT.0",
```
Comment (kbahr) [inline browser-ranking/package.json]: if this works, I'll release this to 0.0.4 and you can update this to a release version

**Review by kbahr — APPROVED** [review]
This lingered long enough in my inbox that the versions got out of date but that's on me :)

**Pull request merged**